### ISSUE TITLE
Error occurred when executing BatchPromptSchedule:

### ISSUE THREAD

**Issue by luoboteqixx** [FizzleDorf/ComfyUI_FizzNodes#109]
I suddenly made a mistake today, can anyone help me?

Error occurred when executing BatchPromptSchedule:

torch.cat(): expected a non-empty list of Tensors

File "G:\ComfyUI\ComfyUI\execution.py", line 317, in execute
output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\ComfyUI\ComfyUI\execution.py", line 192, in get_output_data
return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\ComfyUI\ComfyUI\execution.py", line 169, in _map_node_over_list
process_inputs(input_dict, i)
File "G:\ComfyUI\ComfyUI\execution.py", line 158, in process_inputs
results.append(getattr(obj, func)(**inputs))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\ComfyUI\ComfyUI\custom_nodes\ComfyUI_FizzNodes\ScheduledNodes.py", line 159, in animate
return batch_prompt_schedule(settings, clip)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\ComfyUI\ComfyUI\custom_nodes\ComfyUI_FizzNodes\ScheduleTypes.py", line 49, in batch_prompt_schedule
p = BatchPoolAnimConditioning(pos_cur_prompt, pos_nxt_prompt, weight, clip, settings)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\ComfyUI\ComfyUI\custom_nodes\ComfyUI_FizzNodes\BatchFuncs.py", line 345, in BatchPoolAnimConditioning
final_pooled_output = torch.cat(pooled_out, dim=0)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Close
![FireShot Capture 019 - ComfyUI - 127 0 0 1](https://github.com/user-attachments/assets/5ececa06-05f3-4f18-b8a2-561d7dc78418)

**Comment by luoboteqixx** [comment]
Switch back to version 0e30c12, everything is normal

**Comment by FizzleDorf** [comment]
End frame needs to be set to something not 0

**Comment by FizzleDorf** [comment]
I made a case where if end_frames is 0, it will be max frames. I also added another one so start_frame cannot be equal to or greater than end_frame. try again after pulling.

**Comment by luoboteqixx** [comment]
> I made a case where if end_frames is 0, it will be max frames. I also added another one so start_frame cannot be equal to or greater than end_frame. try again after pulling.

End_frame cannot be set to 0. Changing it to equal the maximum frame will not cause any problems. Thank you

**Comment by thinghesi** [comment]
how to you fix error?

**Comment by mdkberry** [comment]
having this problem. putting a larger numbr in the "end_frame" than is needed by guessing it (e.g. for me, video frame / fps) it then errors with a number of frames you should have put in. using that number seemed to work.
